tensorflow/tensorflow-engine/src/main/java/ai/djl/tensorflow/engine/TfNDArray.java

```diff
@@ -1322,6 +1322,12 @@ public NDArray isNaN() {
     /** {@inheritDoc} */
     @Override
     public NDArray tile(long repeats) {
+        // tf tile doesn't support scalar
+        if (isScalar()) {
+            try (NDArray temp = reshape(1)) {
```

Comment:
Why tile did not fail and affect next operation? Looks like a TF bug?

Reply:
We never test scalar with tile op